RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 2.9.1
```

Comment:
IMO that change is a bit break in compatibility and should be reflected in at least minor number: `2.10.0` or even in major number: `3.0.0`.

Reply:
Actually... I now tried to ensure that it works both under Python 2.7 and Python 3, pls take a look :) Thus, it now merits at most minor version change.